.travis.yml

```diff
@@ -1,10 +1,12 @@
 language: ruby
 sudo: false
 cache: bundler
+install:
+  bundle install
```

Comment:
that breaks bundler caching ... I'll take a look later ... seems weird that it did not work out of the box ...